css/style.css

```diff
@@ -0,0 +1,30 @@
+/* player.js Volume Slider */
+input[type="range" i].volume
```

Comment:
is `i` a typo? The others selectors don't have this, and I'm not sure if that's valid attribute selector syntax. The style _does_ appear to apply to the element though, it renders as upright so that `rotate(270deg)` is definitely happening.

Reply:
@emcniece I am not actually sure why it's there but I found it on a page and it seemed to work so i'll keep it for now